core/Plugin/Controller.php

```diff
@@ -457,26 +457,29 @@ protected function getLastUnitGraphAcrossPlugins(
                 }
             }
         }
-
+    
         // initialize the graph and load the data
         $view = $this->getLastUnitGraph($currentModuleName, $currentControllerAction, $apiMethod);
-
+    
         if ($columnsToDisplay !== false) {
             $view->config->columns_to_display = $columnsToDisplay;
         }
-
+    
         if (property_exists($view->config, 'selectable_columns')) {
-            $view->config->selectable_columns = array_merge($view->config->selectable_columns ? : array(), $selectableColumns);
+            $view->config->selectable_columns = array_merge($view->config->selectable_columns ?: array(), $selectableColumns);
         }
-
+    
         $view->config->translations += $translations;
-
-        if ($reportDocumentation) {
-            $view->config->documentation = $reportDocumentation;
+    
+        // Add fallback for missing documentation
+        if (empty($reportDocumentation)) {
+            $reportDocumentation = "No documentation available for this report.";
```

Comment:
Can you please make this use the translation system so that users using Matomo in other languages can benefit from a translation?